AttitudeManager/Src/AM.cpp

```diff
@@ -1,3 +1,26 @@
 //
 // Created by Anthony Luo on 2022-10-12.
 //
+#include "AM.hpp"
+
+namespace AM {
+
+void AttitudeManager::runControlLoopIteration(
+    AttitudeManagerInput instructions) {
+    // Process Instructions
+
+    // Run Control Algorithms
+    for (auto controller : controller_interfaces) {
```

Comment:
is auto necessary in this case?

Reply:
I mean... it is the deceleration of `controller` so we do need to declare a type. Best practice is to use `auto` iirc

Reply:
whether to use auto or to declare a type is a debate amongst all lol, I think whatever it is we should set a standard of what we claim is an acceptable use of auto and keep the code consistent for when we can

Reply:
I think auto is ok, but I've been told that if I use auto I should leave a comment saying what the type is right above, but also in rtlandia auto is funky...

If you think it is OK in this instance, then it should be ok.

Barr C says auto is:
> Reasoning: The auto keyword is an unnecessary historical feature of the language. The register keyword presumes the programmer is smarter than the compiler. There is no compelling reason to use either of these keywords in modern programming practice.